client/network_disconnect_test.go

```diff
@@ -0,0 +1,64 @@
+package client
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/docker/engine-api/client/transport"
+	"github.com/docker/engine-api/types"
```

Comment:
A little bit of a tangent, but do you know why NetworkDisconnect is in type/types.go instead of types/network/network.go ?  Seems like putting all struct of type XXX should go into the types/XXX/XXX.go file, no?